app/gui/src/providers/session/constants.ts

```diff
@@ -0,0 +1,3 @@
+/** @file Constants for the session provider. */
+
+export const LOGOUT_EVENT = 'enso-logout'
```

Comment:
Here I think it would be nicer to have it just on top of `session.ts` file.

Reply:
on paper that sounds ok but i would like to avoid `session.ts` itself bringin in unnecessary dependencies to anything that uses `LOGOUT_EVENT` (e.g. `zustand.ts`)